### DISCUSSION TITLE
essay/sdzqr3/

### DISCUSSION THREAD

**Discussion by giscus-comment** [wztlink1013/comment#57] [General]
# essay/sdzqr3/

上计算方法课的那个老师，我严重怀疑他有山东口音，他讲话，一开始简直听不清，可能是我还不习惯山东话的缘故吧~可就是在那样的课堂里面，在那些个周里面，我真的是潜移默化般地学到了许多到许多……大三上学期选了一门计算方法这门课，当时选的人不太多，四十几个吧，当时都是说这门课太难了，需要用到许多之前学习的高数

https://www.wztlink1013.com/essay/sdzqr3/

**Comment by zhanjiahui** [comment]
是圣涛吗！我觉得他是济宁口音哈哈哈哈

**Reply by wztlink1013** [reply]
嗯嗯，对(＠_＠;)
